addons/test_mail_full/tests/test_portal.py

```diff
@@ -105,10 +105,9 @@ def test_portal_avatar_with_hash_pid(self):
             url=post_url,
             json={
                 'params': {
-                    'csrf_token': http.Request.csrf_token(self),
```

Comment:
Just to known, why can `csrf_token` be removed from all those tests?

Reply:
[CSRF](https://github.com/odoo/odoo/blob/c9c58b1d54ad411d9b17914ebf0a6de9273b38dc/odoo/http.py#L736): Enabled by default for ``'http'``-type requests, disabled by default for ``'json'``-type requests

The `type` here is `json`.